services/movie_session.py

```diff
@@ -0,0 +1,50 @@
+import datetime
+
+from db.models import MovieSession, Movie, CinemaHall
+
+
+def create_movie_session(
+        movie_show_time: str,
+        movie_id: int,
+        cinema_hall_id: int
+):
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie=Movie.objects.get(id=movie_id),
+        cinema_hall=CinemaHall.objects.get(id=cinema_hall_id)

```

Comment:
you can assign movie and cinema_hall through id and don't make unnecessary queries